core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1357,7 +1345,6 @@ class DAGScheduler(
       logDebug("Additional executor lost message for " + execId +
                "(epoch " + currentEpoch + ")")
     }
```

Comment:
This appears to be a non-issue, because we handle lost shuffle output separately, when we get a FetchFailure from a task that tries to fetch the output.